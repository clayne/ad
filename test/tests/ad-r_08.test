ad | -r | separator.bin | | 0
