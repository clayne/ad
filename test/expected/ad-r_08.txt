Waldo..........
XXXXXXXXXXXXXXXX
 Waldo.........
  Waldo.......
XXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX
   Waldo.......
XXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX
    Waldo......
     Waldo....
XXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX
      Waldo....
